### DISCUSSION TITLE
Website exlint.io is unreachable

### DISCUSSION THREAD

**Discussion by mytlogos** [Exlint/cli#547] [Q&A] [answered]
Neither exlint.io nor docs.exlint.io are reachable.
There are no DNS entries available for these domains.
Are these domains outdated or is this a different problem?
Is there any website (homepage/docs) available?

**Comment by roy-shiloh** [comment, marked as answer]
Hi Exlint is not maintained anymore :)

**Reply by mytlogos** [reply]
Could you document that anywhere on the github repos, org or/and remove these old links?

**Reply by roy-shiloh** [reply]
Sure, thanks.
